### PR TITLE
feat song vete de una vez added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 03/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        Que mi corazón está cansado y ya no puede más
+        Ya perdí todo el honor, llegas y matas lo poco
+        De cordura que me queda y no me dejas avanzar
+        Vete de una vez, que me haces daño
+        ¿No ves que mi alma tiene un límite para el dolor?
+        Yo te di todo mi amor, no lo quisiste y ahora
+        Debes aprender a estar sin mí
+      </h5>
+    </section>
+    <div></div>
```
Comment (andysotocruz) [inline 03/index.html:73]: why is there an empty div?

File: 03/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <footer>
+      <h6>
```
Comment (andysotocruz) [inline 03/index.html:75]: why are there p tags inside an h6?

**Review by andysotocruz — COMMENTED** [review]
where is the nav?

File: 03/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Letra de la canción 'Vete de una vez' de Daniela Spalla">
+  <meta name="keywords" content="daniela, spalla, vete, vez, cancion">
```
Comment (andysotocruz) [inline 03/index.html:7]: there is an error in the content

File: 03/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        Ya perdí todo el honor, llegas y matas lo poco
+        De cordura que me queda y no me dejas avanzar
+        Vete de una vez, que me haces daño
+        ¿No ves que mi alma tiene un límite para el dolor?
+        Yo te di todo mi amor, no lo quisiste y ahora
+        Debes aprender a estar sin mí
+      </h5>
+    </section>
+    <div></div>
+    <footer>
```
Comment (eamzea) [inline 03/index.html:74]: Why main is not the main tag in this document?
If you have more doubts, take a look at this link https://dev.to/sapardo/etiquetas-semanticas-de-html5-3j7m

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by andysotocruz — COMMENTED** [review]
again, where is the nav?

**Review by andysotocruz — DISMISSED** [review]

File: 03/index.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  <title>Vete de una vez - Daniela Spalla</title>
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <h1>Vete de una vez</h1>
+      </nav>
+      <h2>Daniela Spalla</h2>
+    </header>
```
Comment (andysotocruz) [inline 03/index.html:17]: Indentation.

**Review by soyedgarzea — CHANGES_REQUESTED** [review]
Conflicts in the branch

File: 03/index.html [hunk 6]
```diff
@@ -1,82 +1,154 @@
...
-            <h6>
-                © Kobalt Music Publishing Ltd., Sony/ATV Music Publishing LLC, Universal Music Publishing Group 
-            </h6>
-        </footer>
-    </main>
-</body>
-</html>
\ No newline at end of file
+    <header>
+      <h1>lovely</h1>
```
Comment (eamzea) [inline 03/index.html:87]: There should be only one h1 inside the html file

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 03/index.html [hunk 7]
```diff
@@ -1,82 +1,147 @@
...
+        De cordura que me queda y no me dejas avanzar
+        Vete de una vez, que me haces daño
+        ¿No ves que mi alma tiene un límite para el dolor?
+        Yo te di todo mi amor, no lo quisiste y ahora
+        Debes aprender a estar sin mí
+      </h5>
+    </section>
+  <article>
+    <h4>Outro</h4>
+  </article>
```
Comment (eamzea) [inline 03/index.html:50]: Looks like here was a indentation issue

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**